src/hdf5/atms_netcdf_hdf5_2ioda.py

```diff
@@ -312,6 +286,29 @@ def concat_obs_dict(obs_data, append_obs_data):
             print("WARNING: ", gv_key, " is missing from append_obs_data dictionary")
 
 
+def set_metadata_attributes(VarAttrs):
+    VarAttrs[('sensorZenithAngle', 'MetaData')]['units'] = 'degree'
+    VarAttrs[('sensorViewAngle', 'MetaData')]['units'] = 'degree'
+    VarAttrs[('solarZenithAngle', 'MetaData')]['units'] = 'degree'
+    VarAttrs[('sensorAzimuthAngle', 'MetaData')]['units'] = 'degree'
+    VarAttrs[('solarAzimuthAngle', 'MetaData')]['units'] = 'degree'
+
+    VarAttrs[('dateTime', 'MetaData')]['units'] = "seconds since 1970-01-01T00:00:00Z"
```

Comment:
I previously commented on this but this units does not align with a string; only when dateTime is filled with integers

Reply:
yup